hazelcast-jet-distribution/src/root/config/jvm-client.options

```diff
@@ -0,0 +1,12 @@
+# JVM Configuration for bin/jet command
+
+# One option per line
+# Lines starting with # and empty lines are ignored
+
```

Comment:
we should add some more, for example to use g1gc if you're using Java 8 and also options to enable GC logging